### ISSUE TITLE
[Bug, Report], el-tooltip, with, el-button, failed, to, popup, when,, using, Vue, 2.4.1

### ISSUE THREAD

**Issue by zerob13** [ElemeFE/element#5912]
### Element UI version
1.3.7

### OS/Browsers version
macOS 10.13| Ubuntu 16.04 LTS / Chrome 60.0.3122.66 beta

### Vue version
2.4.1

### Reproduction Link
[https://jsfiddle.net/mksfL2sf/1/](https://jsfiddle.net/mksfL2sf/1/)

### Steps to reproduce
Follow code will failed with Vue 2.4.1 ,but works with 2.3.x
 ```
<el-tooltip class="item" effect="dark" content="Top Left 提示文字" placement="right">
      <el-button>无效</el-button>
  </el-tooltip>
```
If you add a div wrap ,the tooltips works again.
 ```
<el-tooltip class="item" effect="dark" content="Top Left 提示文字" placement="right">
  <div>
      <el-button>有效</el-button>
  </div>
  </el-tooltip>
```

### What is Expected?
el-tooltips works with el-button

### What is actually happening?
el-tooltips doesn't work with el-button

**Comment by Leopoldthecoder** [comment]
Fixed in #5916

**Comment by lazybonee** [comment]
Vue version 2.5.3
Element UI version 2.3.9
OS/Browsers version  Windows 10/chrome 67.0.3396.79
El-tooltips don't work with el-button
